Fortran/gfortran/CMakeLists.txt

```diff
@@ -63,18 +63,62 @@ set(TEST_SUITE_FORTRAN_ISO_C_HEADER_DIR "" CACHE STRING
 # be because they are currently unsupported and might eventually be supported
 # or because they are GCC-specific and will never be supported.
 set(FLANG_ERRORING_FFLAGS
+  # The "flags" that do not start with a hyphen (-) are actually arguments to
+  # some other flag but use the "-flag value" syntax (as opposed to the
+  # "-flag=value" syntax). Since we don't actually parse the command line, they
+  # are treated as if they were "flags" .
+  analyzer-max-svalue-depth=0
+  ggc-min-expand=0
+  ggc-min-heapsize=0
+  iv-max-considered-uses=2
+  large-stack-frame=4000
   max-completely-peel-loop-nest-depth=1
+  max-completely-peeled-insns=0
+  max-completely-peeled-insns=200
```

Comment:
Why does `max-completely-peeled-insns` appear twice?
Is `max-completely-peeled-insns` only not supported with values 0 and 200?
If not, shouldn't it (and the other flags) always be ignored, independently of its value?

Reply:
The parser for the DejaGNU attributes in `utils/update-test-config.py` does not currently deal with command line flags correctly. It simply splits the annotation on whitespace and treats each substring as a "flag". It therefore cannot tell the difference between `--flag=val1` and `--flag=val2`. There are other consequences of this. At the start of the list there are entries which do not begin with `-`. This is because they are written in the annotations as `--flag val` and the parser does not know that this is equivalent to `--flag=val`. 

Making the parser a bit smarter about flags is on my to-do list.

Reply:
Ok. Then maybe a TODO comment could be added, briefly explaining the current limitations of the parser, and that later this blacklist could be simplified, by removing the flags' values and duplicates.

Reply:
Done.